### PR TITLE
List jobs added to k8s client

### PR DESCRIPTION
**Pull request by nityanandagohain**

### CODE REVIEW

File: pkg/client/kube.go [hunk 1]
```diff
@@ -97,3 +97,35 @@ func (k *K8s) ListPods(namespace string) ([]PodInfo, error) {
...
+		totalPods := job.Status.Active + job.Status.Succeeded + job.Status.Failed
+
+		var duration float64
+		if job.Status.CompletionTime != nil {
+			duration = job.Status.CompletionTime.Time.Sub(job.ObjectMeta.CreationTimestamp.Time).Seconds()
+		}
+
+		p := JobInfo{
+			Name:        job.Name,
+			Completions: fmt.Sprintf("%v/%v", job.Status.Succeeded, totalPods),
```
Comment (GnaneshKunal) [inline pkg/client/kube.go]: Let's create additional fields for total succeeded jobs and total pods and let the render layer decide oh the format of `"completions". If in the future, we had to know the total number of succeeded jobs, the stringified implementation will come in handy.

File: pkg/client/kube.go [hunk 2]
```diff
@@ -97,3 +97,35 @@ func (k *K8s) ListPods(namespace string) ([]PodInfo, error) {
...
+			Duration:    duration,
+			CreatedAt:   job.ObjectMeta.CreationTimestamp.Time,
```
Comment (GnaneshKunal) [inline pkg/client/kube.go]: `Age` Field is not used.

**Review by GnaneshKunal — CHANGES_REQUESTED** [review]

File: pkg/client/kube.go (hunk 1 above)
Comment (nityanandagohain) [reply pkg/client/kube.go]: Cool.

File: pkg/client/kube.go (hunk 2 above)
Comment (nityanandagohain) [reply pkg/client/kube.go]: Have added CompletedAt. Is Age required now?

File: pkg/client/kube.go (hunk 1 above)
Comment (nityanandagohain) [reply pkg/client/kube.go]: Have made the changes

**Review by GnaneshKunal — APPROVED** [review]
Since `Age` is not required, we can remove it when revisiting the `struct` later. LGTM right now.

**Pull request merged**